.ci/ct.yaml

```diff
@@ -0,0 +1,5 @@
+helm-extra-args: --timeout 7200
```

Comment:
could you please add a license header to this file?
```
# Copyright (c) 2019 Contributors to the Eclipse Foundation
#
# See the NOTICE file(s) distributed with this work for additional
# information regarding copyright ownership.
#
# This program and the accompanying materials are made available under the
# terms of the Eclipse Public License 2.0 which is available at
# http://www.eclipse.org/legal/epl-2.0
#
# SPDX-License-Identifier: EPL-2.0
---
```

Reply:
added